Microsoft.Toolkit.Uwp.UI.Controls.Input/RangeSelector/RangeSelector.Properties.cs

```diff
@@ -0,0 +1,275 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// RangeSelector is a "double slider" control for range values.
+    /// </summary>
+    public partial class RangeSelector : Control
+    {
+        /// <summary>
+        /// Identifies the Minimum dependency property.
+        /// </summary>
+        public static readonly DependencyProperty MinimumProperty =
+            DependencyProperty.Register(
+                nameof(Minimum),
+                typeof(double),
+                typeof(RangeSelector),
+                new PropertyMetadata(DefaultMinimum, MinimumChangedCallback));
+
+        /// <summary>
+        /// Identifies the Maximum dependency property.
+        /// </summary>
+        public static readonly DependencyProperty MaximumProperty =
+            DependencyProperty.Register(
+                nameof(Maximum),
+                typeof(double),
+                typeof(RangeSelector),
+                new PropertyMetadata(DefaultMaximum, MaximumChangedCallback));
+
+        /// <summary>
+        /// Identifies the RangeMin dependency property.
+        /// </summary>
+        public static readonly DependencyProperty RangeMinProperty =
+            DependencyProperty.Register(
+                nameof(RangeMin),
+                typeof(double),
+                typeof(RangeSelector),
+                new PropertyMetadata(DefaultMinimum, RangeMinChangedCallback));
+
+        /// <summary>
+        /// Identifies the RangeMax dependency property.
+        /// </summary>
+        public static readonly DependencyProperty RangeMaxProperty =
```

Comment:
We should still update these DP to `RangeStartProperty` and `RangeEndProperty` along with the other backing properties - That effects the sample app as well, you can grab my sample app condensing bits from the refactor branch as well.

Reply:
Got the commit